NEWS

```diff
@@ -1,5 +1,7 @@
 # Machinery Release Notes
 
+* Offer files containing '%' and other special characters for download
```

Comment:
"Offer files" sounds misleading, I would write something like:
"Files with special characters are now downloadable (..)" or
"File names are now escaped in HTML view (...)"